guides/source/ruby_on_rails_guides_guidelines.md

```diff
@@ -108,10 +108,7 @@ HTML Guides
 -----------
 
 Before generating the guides, make sure that you have the latest version of
-Bundler installed on your system. You can find the latest Bundler version
-[here](https://rubygems.org/gems/bundler). As of this writing, it's v1.17.1.
-
-To install the latest version of Bundler, run `gem install bundler`.
+Bundler installed on your system. To install the latest version of Bundler, run `gem install bundler`.
```

Comment:
Worth mentioning `gem update bundler`?

Reply:
Good suggestion @sambostock. Just added a line for that.